frontend/src/types/oppgavebehandling-params.ts

```diff
@@ -22,8 +22,9 @@ export type ITilknyttDocumentParams = IDocumentReference & IOppgavebehandlingBas
 
 export interface IMedunderskrivereParams {
   navIdent: string;
-  ytelseId: string;
+  ytelse: string;
```

Comment:
Hvorfor rename denne?

Reply:
Siden det er det propen heter i API-et - så slapp jeg å rename tilbake i RTKQ-fila. Men kanskje det er mer tydelig med `ytelseId`, ja?

Reply:
Samme som APIet vil ha. Da burde vi be de om å endre, slik at det forblir konsekvent.